_posts/en/newsletters/2021-02-10-newsletter.md

```diff
@@ -0,0 +1,231 @@
+     pre-segwit compact blocks to peers is no longer useful; without the
+     witnesses, peers are unable to validate the consensus rules on
+     them."
+
+  a5link="https://bitcoincore.reviews/20799#l-104"
+
+  q6="Practically speaking, how can version 1 support be dropped?"
+  a6="By ignoring incoming `sendcmpct` messages with version equal to 1, no
+     longer sending `sendcmpct` messages that signal version 1 support, only
+     sending `sendcmpct` messages to `NODE_WITNESS` peers, and responding to
+     `sendcmpct` and `blocktxn` messages with witness-serialized blocks and
+     transactions."
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [HWI #433][] adds support for signing PSBTs with OP_RETURN outputs.
+
+- [Bitcoin Core #19509][] adds per-peer message capture between nodes as well
+  as the ability to produce JSON outputs from those logs.  Using the newly
+  introduced command line argument `capturemessages`, any message
+  that the node sends or receives is logged.  There is a
+  [long history][bitcoin dissector] of tools with similar functionality,
+  but this addition provides a native alternative to less actively
+  maintained options.
+
+- [BIPs #1021][] updates the [BIP8][] soft fork activation mechanism,
+  changing its behavior for nodes that choose to enforce the *lockin on
+  timeout* feature.  Previously, those nodes would reject any block that
+  didn't signal for activation during the mandatory activation period.
+  After this change, the nodes will tolerate up to the maximum number of
+  non-signaling blocks possible that still guarantee activation.  This
+  reduces the number of blocks that may be needlessly rejected and
+  reduces the potential for miscommunication between miners and node
+  operators.
+
+- [BIPs #1020][] updates BIP8 to no longer require signaling during the
+  *locked in* period now that activation can be assured using required
+  signaling in the [recently added][BIPs #950] *must signal* period.
+
+- [Bitcoin Core #20764][] adds additional information to the output
+  produced using `bitcoin-cli -netinfo`.  New details include the number
+  of manually added peers, peers using [BIP152][] "high-bandwidth"
+  (faster) block relay mode, and any peers using the [I2P][] anonymity
+  network (being worked on in a [separate PR][bitcoin core #20685]).
```

Comment:
maybe I2P ("Invisible Internet Project") network

Reply:
I think I2P is sort of like Tor in the sense that the abbreviation of the original name has become the preferred name, so expanding the abbreviation doesn't add any useful information.